packages/nx/src/native/tui/app.rs

```diff
@@ -769,8 +779,25 @@ impl App {
             trace!("{action:?}");
         }
         match &action {
+            Action::StartCommand(_) => {
+                self.console_messenger
+                    .as_ref()
+                    .and_then(|c| c.start_running_tasks());
+            }
+            Action::Tick => {
+                self.console_messenger.as_ref().and_then(|messenger| {
```

Comment:
Should we throttle/debounce in some way? Updating everything on every tick is quite intense, no?

Reply:
Yes. That happens here:
https://github.com/nrwl/nx/pull/31148/files#diff-395a7cc58ef509a48734eb3ffee39299d9af81020dd9115329de1c4404165faeR106-R108